recipes/main/black-pepper-tofu.md

```diff
@@ -0,0 +1,50 @@
+# Black pepper tofu
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-60%25-orange.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1.5hr-orange.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-orange.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**First cooked by:** [@matthewdking](https://github.com/matthewdking/)
+
+**Date:** Week 16 FACN2
+
+**Author of Recipe**: [@matthewdking](https://github.com/matthewdking/)
```

Comment:
A nod to the original author ([Yotam Ottolenghi](https://www.ottolenghi.co.uk/black-pepper-tofu-a-shop)) is always appreciated. I for one think he'd be flattered to be included in such a prestigious cooking forum.

Reply:
Does he have GitHub though???